bin/strata-cli/src/constants.rs

```diff
@@ -4,17 +4,22 @@ use alloy::consensus::constants::ETH_TO_WEI;
 use bdk_wallet::bitcoin::{Amount, Network};
 
 /// Number of blocks after bridge in transaction confirmation that the recovery path can be spent.
-pub const RECOVER_DELAY: u32 = 1008;
+///
+/// 144 is the number of blocks in a day.
+pub const RECOVER_DELAY: u32 = 144;
+
+/// Number of block that we consider a transaction "buried" or final.
+pub const FINALITY_DEPTH: u32 = 6;
 
 /// Number of blocks after which we'll actually attempt recovery. This is mostly to account for any
```

Comment:
Who is "we" here? Isn't the user able to initiate the tx once the timelock is over? Will the wallet deny it?
Or is this that the wallet initiates the take-back automatically once the right time arrives?

Reply:
Clarified the "we" in 2a0a89206c3a47361a5d46ac9e8163f218a15681

This is used internally to create a recovery wallet descriptor to "take back" a DRT after the 144 + 6 blocks `RECOVER_DELAY + FINALITY_DEPTH`.

This is done at deposit time when sending the DRT:

https://github.com/alpenlabs/strata/blob/10dfce977bb4b7d981568612c17652d824206b84/bin/strata-cli/src/cmd/deposit.rs#L92

https://github.com/alpenlabs/strata/blob/10dfce977bb4b7d981568612c17652d824206b84/bin/strata-cli/src/cmd/deposit.rs#L131-L138

And the user can use the `recover` command to take back the unprocessed DRT: https://github.com/alpenlabs/strata/blob/10dfce977bb4b7d981568612c17652d824206b84/bin/strata-cli/src/cmd/recover.rs#L25-L33

Reply:
Thanks, I (tried to) read the code and got red.


>the wallet actually attempts recovery

Will be reading separately what wallet descriptor means, but if I got the gist then it's not that the wallet _attempts_ recovery at this height (that is, the wallet does not initiates txes autonomously), but it enables the take back at this height. Is it so?
If yes, let's change the phrasing a bit, maybe `attempts`-->`enables`.

Reply:
Done in dbc56b692e43ef702849fa480e1de9135f7e25cd